spec/factories.rb

```diff
@@ -91,6 +102,10 @@
       with_location
     end
 
+    factory :cyclist do
+      admin
+    end
+
     trait :admin do
       admin true
     end
```

Comment:
I think you can make `:admin` a subfactory with `:cyclist` then being an alias here if you like the way that looks better. I often find myself using aliases so the declarations in specs are shorter.